app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/landing_page.scala.html

```diff
@@ -83,20 +84,7 @@ <h2 class="govuk-heading-l">@messages(s"$key.before.heading")</h2>
 
 
         <div class="space-document-type">
```

Comment:
@oddbunchmedia I added this class when I was originally designing the pages. It would be good if we can take this out.

Reply:
I can just remove that, it doesn't change how the page renders